docs/DeveloperGuide.md

```diff
@@ -158,6 +158,18 @@ Classes used by multiple components are in the `seedu.address.commons` package.
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Add Transaction 
```

Comment:
maybe it would be good to put the command here as well? like "Add Transaction `addt`". This would make it easier to relate to our commands?

Reply:
ok I've made the change